end-to-end-tests/templates/test_cluster.yml

```diff
@@ -55,6 +55,21 @@ Resources:
         Environment:
         - Name: TIME
           Value: "60"
+  PersistentTask:
+    Type: AWS::ECS::TaskDefinition
+    Properties:
+      Family:
+        Fn::Join: ['', [{ Ref: 'AWS::StackName' }, -persistent-task]]
+      ContainerDefinitions:
+      - Name: sleep
+        Cpu: '10'
+        Memory: '300'
+        Essential: 'true'
+        Image: 'alpine:latest'
+        Command: ['/bin/sh', '-c', 'trap : TERM INT; sleep infinity & wait']
```

Comment:
Looks like this task just sleeps forever, so it will never end. Why do we want this behavior for this task? I think the transient task definition also works for testing the update environment.

Reply:
Primarily, I justified this to myself by reasoning that the task should not accidentally exit (because its timeout expired) before it can be upgraded explicitly by the scheduler.

However, mainly I just needed another task definition that could be referenced from the tests.

Reply:
Makes sense! Actually, a daemon task should always be running in the background, so the new task that you added is closer to a real daemon task.